frontend/elements/src/components/icons/ExclamationMark.tsx

```diff
@@ -1,14 +1,26 @@
 import * as preact from "preact";
-
 import styles from "./styles.sass";
+import { IconProps } from "./Icon";
+import cx from "classnames";
 
-const ExclamationMark = () => {
+const ExclamationMark = ({ size, secondary, style, disabled }: IconProps) => {
   return (
-    <div className={styles.exclamationMark}>
-      <div className={styles.circle} />
-      <div className={styles.stem} />
-      <div className={styles.dot} />
-    </div>
+    <svg
```

Comment:
The icon looks like an info icon containing an "i" instead of an exclamation mark "!".